pkg/projects/Microsoft.NETCore.UniversalWindowsPlatform/Microsoft.NETCore.UniversalWindowsPlatform.pkgproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
 
   <PropertyGroup>
-    <Version>5.2.3</Version>
+    <Version>5.2.4-beta</Version>
```

Comment:
@gkhanna79 We want a prerelease nuget package so we appended beta to the 5.2.4 version. Look good?